src/test/java/com/google/jenkins/plugins/storage/client/ClientFactoryTest.java

```diff
@@ -46,14 +45,15 @@ public void defaultTransport() throws Exception {
     JsonServiceAccountConfig serviceAccountConfig = new JsonServiceAccountConfig();
     serviceAccountConfig.setSecretJsonKey(bytes);
     assertNotNull(serviceAccountConfig.getAccountId());
-    Credentials c =
-        (Credentials) new GoogleRobotPrivateKeyCredentials(ACCOUNT_ID, serviceAccountConfig, null);
+    GoogleRobotPrivateKeyCredentials c =
```

Comment:
is this not the deprecated constructor - should this not be using the non deprecated one, and update the dependency to the newer version of this plugin and pass in the credentialID that you want

Reply:
Hi @jtnord :wave: 

Sadly I cannot push to @Riliane 's PR, so I took care of your comment in a separate PR: https://github.com/jenkinsci/google-storage-plugin/pull/266

Could you please give it a look?

Thanks a lot!